experimental/PropertyDDS/examples/property-inspector/webpack.config.js

```diff
@@ -45,6 +45,9 @@ module.exports = (env) => {
 			new webpack.DefinePlugin({
 				"process.env.NODE_DEBUG": undefined,
 			}),
+			new webpack.DefinePlugin({
+				"process?.env?.NODE_NAME": undefined,
```

Comment:
these question marks belong in server/routerlicious/packages/lambdas/src/alfred/index.ts or this define belongs here. Pick one.

Reply:
Same for the other file

Reply:
Also probably better to use one define plugin instance with both definitions instead of two separate ones.

Also in this case they likely collide, so you probably want:
```
new webpack.DefinePlugin({
				"process.env": {NODE_DEBUG: undefined, NODE_NAME: undefined}
}),
```

Docs are over at https://webpack.js.org/plugins/define-plugin/